### PR TITLE
check for undefined value in setColorProperty and resetColorProperty …

### PR DESCRIPTION
**Pull request by tsonevn**
Checking for using `undefined` values for both methods
* `setColorProperty`
* `resetColorProperty`

### CODE REVIEW

**Comment by dtopuzov** [comment]
run ci

**Comment by hshristov** [comment]
run ci

File: tns-core-modules/ui/text-base/text-base-styler.android.ts [hunk 1]
```diff
@@ -21,11 +21,15 @@ export class TextBaseStyler implements style.Styler {
...
     private static setColorProperty(view: view.View, newValue: any) {
-        (<android.widget.TextView>view._nativeView).setTextColor(newValue);
+        if(newValue){
+            (<android.widget.TextView>view._nativeView).setTextColor(newValue);
+        }
     }
 
     private static resetColorProperty(view: view.View, nativeValue: any) {
-        (<android.widget.TextView>view._nativeView).setTextColor(nativeValue);
+        if(nativeValue){
```
Comment (vakrilov) [inline tns-core-modules/ui/text-base/text-base-styler.android.ts]: This if will skip value setting if `nativeValue` is 0(number).

File: tns-core-modules/ui/text-base/text-base-styler.android.ts [hunk 2]
```diff
@@ -21,11 +21,15 @@ export class TextBaseStyler implements style.Styler {
 
     // color
     private static setColorProperty(view: view.View, newValue: any) {
-        (<android.widget.TextView>view._nativeView).setTextColor(newValue);
+        if(newValue){
```
Comment (vakrilov) [inline tns-core-modules/ui/text-base/text-base-styler.android.ts]: This if will skip value setting if nativeValue is 0(number).

**Pull request closed without merging**